sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -0,0 +1,59 @@
+from typing import Sequence
+from collections import defaultdict
+from datetime import datetime
+from azure.eventhub.aio import CheckpointStore, EventHubConsumerClient
```

Comment:
@chradek, good catch! this should be sync:
```suggestion
from azure.eventhub import CheckpointStore, EventHubConsumerClient
```
The async version is under a different directory completely, which would be at `sdk/eventhub/azure-checkpointstoretable-aio`. 
As for which is used more, I'm not sure whether we have data on that, but I'll check with Adam and get back to you!

Reply:
Update: so it looks like the async version is used more. I still think sync should be the focus since understanding/implementing/testing is easier with sync and async is just copy-pasting :)